src/NServiceBus.Core/Encryption/ConfigureRijndaelEncryptionService.cs

```diff
@@ -26,71 +30,103 @@ public static void RijndaelEncryptionService(this BusConfiguration config)
                 var section = context.Build<Configure>()
```

Comment:
@andreasohlund What do you mean? There is the `Encryptor` feature that does the registration of the behaviors.